algorithms/tree/bst/depth_sum.py

```diff
@@ -47,7 +47,7 @@ def recur_depth_sum(root, n):
 """
 
 class TestSuite(unittest.TestCase):
-    def setUp(self):
+    def setup(self):
```

Comment:
actually `setUp` is the abstract method of `unittest.TestCase`, and we override it here. So, please it back to `setUp`. Thanks.